suite-native/module-onboarding/src/screens/WelcomeScreen.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { ReactNode } from 'react';
+
+import { useNavigation } from '@react-navigation/native';
+
+import { Box, Text, Image } from '@suite-native/atoms';
+import {
+    OnboardingStackParamList,
+    OnboardingStackRoutes,
+    Screen,
+    StackNavigationProps,
+} from '@suite-native/navigation';
+import { Icon } from '@trezor/icons';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { Color } from '@trezor/theme';
+
+import { GetTrezor } from '../components/GetTrezor';
+import { GetTrezorButton } from '../components/GetTrezorButton';
+
+const titleStyle = prepareNativeStyle(utils => ({
+    maxWidth: '60%',
+    textAlign: 'center',
+    marginBottom: 12,
+    ...utils.typography.titleMedium,
+}));
+
+const imageStyle = prepareNativeStyle(() => ({
+    position: 'absolute',
+    left: 0,
+    right: 0,
+    top: 0,
+}));
+
+const TitleText = ({ children, color }: { children: ReactNode; color: Color }) => (
+    <Text variant="titleMedium" color={color} style={{ textAlign: 'center' }}>
+        {children}
+    </Text>
+);
+
+export const WelcomeScreen = () => {
+    const navigation =
+        useNavigation<
+            StackNavigationProps<OnboardingStackParamList, OnboardingStackRoutes.Welcome>
+        >();
+    const { applyStyle } = useNativeStyles();
+
+    const handleRedirect = () => {
+        navigation.navigate(OnboardingStackRoutes.TrackBalances);
+    };
+
+    return (
+        <>
+            <Screen>
+                <Box flex={1} justifyContent="space-between" alignItems="center">
+                    <Box flex={2} />
+                    <Box alignItems="center" flex={1}>
+                        <Box alignItems="center">
+                            <Box marginBottom="large">
+                                <Icon size="large" name="trezor" color="backgroundPrimaryDefault" />
+                            </Box>
+                            <Text style={applyStyle(titleStyle)}>
+                                <TitleText color="textDefault">Welcome to</TitleText>
+                                <TitleText color="textSecondaryHighlight"> Trezor Suite</TitleText>
+                                <TitleText color="textSubdued"> Lite</TitleText>
+                            </Text>
+                        </Box>
+                        <Text color="textSubdued">Simple and secure portfolio tracker</Text>
+                    </Box>
+                    <GetTrezor>
+                        <GetTrezorButton redirectTarget={handleRedirect} />
+                    </GetTrezor>
+                </Box>
+            </Screen>
+            <Box style={applyStyle(imageStyle)}>
```

Comment:
This button is called `GetTrezorButton` but it is used for navigation to next screen, not for redirect to Trezor.io. Why did you choose this naming? Also no reason to render it inside of the GetTrezor component, should be rendered underneath it. 

I guess no special component is needed for this. Could be normal atom `<Button onPress={handleOnPress} >Get started<Button />`, same as you did in `AnalyticsConsentScreen`.

Same comment applies for all the other screens.

Reply:
Yeah, good point. I created it when I used this component diferently and then didn't refactor it back. I guess it's overcomplicated for nothing, so fixed here [411b2ef](https://github.com/trezor/trezor-suite/pull/7889/commits/411b2ef4d48df39092779f8f118b3b7d57b00cc0)